Could we ignore the vendor folder?

I noticed that the vendor folder is included in the results from interfacer. The vendor folder is a little beyond my control and I don't think that interfacer should check the vendor folder.


For example, one of my projects vendors github.com/stretchr/testify and I see this error from interfacer:
```bash
interfacer ./...
vendor/github.com/stretchr/testify/mock/mock.go:276:36: t can be assert.TestingT
```

Perhaps it's a good idea to update the README to provide a better example of doing:

```
$ interfacer $(go list ./... | grep -v /vendor/)
foo.go:10:19: f can be io.Reader
```

Than the current `interfacer ./...`. Doing that is a really bad idea unless there are no vendor folders, so why does the README recommend it?

--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ func ProcessInput(f *os.File) error {
 ```
 
 ```sh
-$ interfacer ./...
+$ interfacer $(go list ./... | grep -v /vendor/)
 foo.go:10:19: f can be io.Reader
 ```
 

